discovery-provider/src/api/v1/users.py

```diff
@@ -569,3 +569,42 @@ def get(self, user_id):
         users = get_followees_for_user(args)
         users = list(map(extend_user, users))
         return success_response(users)
+
+top_genre_users_route_parser = reqparse.RequestParser()
+top_genre_users_route_parser.add_argument('genre', required=False, action='append')
+top_genre_users_route_parser.add_argument('limit', required=False, type=int)
+top_genre_users_route_parser.add_argument('offset', required=False, type=int)
+top_genre_users_response = make_response("top_genre_users_response", full_ns, fields.List(fields.Nested(user_model_full)))
```

Comment:
if you wanted, you could just use the full_user_list_response instead of creating a new make_response for each of these